ModularTegustation/tegu_items/apostle_items.dm

```diff
@@ -0,0 +1,307 @@
+/obj/item/dark_bible
+	name = "dark bible"
+	desc = "The darkest of hearts, obtained from the beast made of light. Allows you to perform a ritual to allow other people wield apostle weaponry."
+	icon = 'ModularTegustation/Teguicons/teguitems.dmi'
+	icon_state = "darkbible"
+	item_flags = NOBLUDGEON
+	w_class = WEIGHT_CLASS_NORMAL
+	var/uses = 6
+	var/audio_cd // To prevent sound abuse
+
+/obj/item/dark_bible/afterattack(atom/target, mob/user, proximity_flag)
+	. = ..()
+	if(proximity_flag && ishuman(target))
+		if(user != target)
+			if(uses > 0 && audio_cd < world.time)
+				var/mob/living/carbon/human/H = target
+				if("apostle" in H.faction)
+					to_chat(user, "<span class='info'>[H] is already empowered by dark light.</span>")
+					return
+				user.visible_message("<span class='info'>[user] puts hand on [H]'s shoulder, with [src] in the other hand and starts murmuring something.</span>", \
+				"<span class='warning'>You begin spelling the prayer to grant power to [H].</span>", \
+				"<span class='hear'>You can hear some sort of a prayer nearby.</span>")
+				audio_cd = (world.time + 15 SECONDS)
+				playsound(src, 'ModularTegustation/Tegusounds/apostle/antagonist/whisper.ogg', 50, 1)
+				if(!do_after(user, 100))
+					return
+				uses -= 1
+				H.faction |= "apostle"
+				to_chat(user, "<span class='info'>You finish the prayer, and suddenly, [H] starts to glow witn an ominous light.</span>")
+				to_chat(H, "<span class='info'>As soon as [user] finishes reading the prayer, you start glowing with an ominous light.</span>")
+				H.set_light_color(COLOR_RED_LIGHT)
+				H.set_light(2)
+			else
+				to_chat(user, "<span class='info'>Pages in [src] seem blank. Perhaps there will be a use for it later?</span>")
+		else
+			to_chat(user, "<span class='info'>You can't offer a prayer for yourself!</span>")
+
+/obj/item/clothing/suit/armor/apostle
+	name = "paradise lost"
+	desc = "Your armor, to protect the holy one."
+	worn_icon = 'ModularTegustation/Teguicons/teguclothing_worn.dmi'
+	icon_state = "apostlearmor"
+	item_flags = DROPDEL
+	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	armor = list(MELEE = 80, BULLET = 60, LASER = 60, ENERGY = 80, BOMB = 100, BIO = 100, RAD = 90, FIRE = 100, ACID = 95, WOUND = 25)
```

Comment:
Does this not seem too strong? Pretty sure this is ERT-level.

Reply:
It's basically the same thing as summoning Nar'Sie, albeit with a small chance of winning.